aria-practices.html

```diff
@@ -3697,6 +3697,1068 @@ <h5>Examples</h5>
+ &lt;thead>
+  &lt;tr>
+   &lt;th>
+   &lt;th>Okinawa, 1949
+   &lt;th>Japan, 1950
+ &lt;tbody>
+  &lt;tr>
+   &lt;th>Total calories
+   &lt;td>1785
+   &lt;td>2068
+
+  [...]
+
+&lt;/table></code></pre>
+          <p>Similarly, the HTML <code>figure</code> element can be given a caption using the <code>figcaption</code> element. The caption can appear before or after the figure, but it is more common for figures to have the caption after.</p>
+          <pre><code>&lt;figure>
+ &lt;img alt="Painting of a person walking in a desert." src="Hole_JesusalDesierto.jpg">
+ &lt;figcaption>Jesus entering the desert as imagined by William Hole, 1908&lt;/figcaption>
+&lt;/figure></code></pre>
+        </section>
+
+        <section id="naming_with_fallback">
+          <h5>Fallback Names Derived from Titles and Placeholders</h5>
+          <p>To-do: finish writing this section. Needs to cover title, placeholder, and aria-placeholder.</p>
+          <p>
+            For the HTML <code>input</code> element, the <code>placeholder</code> attribute is used as a fallback labeling mechanism if nothing else results in a label. For an <code>input</code> element, it is better to use a <code>label</code> element, since it does not disappear visually when the user focuses the form control.
+          </p>
+        </section>
+      </section>
+
+      <section id="naming_effectively">
+        <h4>Composing Effective and User-friendly Accessible Names</h4>
+        <p>
+          For assistive technology users, especially screen reader users, the quality of accessible names is one of the most  significant contributors to usability.
+          Names that do not provide enough information reduce users' effectiveness while names that are too long reduce efficiency.
+          And, names that are difficult to understand reduce effectiveness, efficiency, and enjoyment.
+        </p>
+        <p>The following guidelines provide a starting point for crafting user friendly names.</p>
+        <ul>
+          <li>
+            Convey function or purpose, not form.
+            For example, if an icon that looks like the letter <q>X</q> closes a dialog, name it <q>Close</q>, not <q>X</q>.
+            Similarly, if a set of navigation links in the left side bar navigate among the product pages in a shopping site, name the navigation region <q>Product</q>, not <q>Left</q>.
+          </li>
+          <li>
+            Put the most distinguishing and important words first.
+            Often, for interactive elements that perform an action, this means a verb is the first word.
+            For instance, if a list of contacts displays a separate set of edit, delete, and more actions buttons for each contact, <q>Actions for John Doe</q> is better than <q>John Doe actions</q> because it distinguishes the actions menu button from the other buttons as well from the element that opens John Doe's contact card.
```

Comment:
This sentence could be a bit confusing, i.e.
- is the menu button's name "Actions", "More Actions", or just "More..."?
- is this sentence about distinguishing John Doe buttons from each other, or distinguishing John's Actions button from other Action buttons?

It might be clearer to give more detail? Something like:
```
For instance, if a list of contacts displays a separate set of 'Edit', 'Delete', and 'Actions...' buttons for each contact, then <q>Edit John Doe</q>, <q>Delete John Doe</q>, and <q>Actions for John Doe</q> are better than <q>John Doe edit</q>, <q>John Doe delete</q>, or <q>John Doe actions</q> because placing the verb first distinguishes the buttons from each other as well from the element that opens John Doe's contact card.
```

Reply:
Thank you for the suggested replacement. I reworded similarly.